docs/next/guides/building-and-testing/testing.md

```diff
@@ -27,8 +27,12 @@ To run the Jasmine tests in your browser, go to the following pages:
 
 ## Testing with NPM (CLI)
 
-To run Jasmine tests in your command line (using Puppeteer), first install all necessary dependencies by executing `npm install`. After the `NPM` finishes fetching required modules your workspace is ready to test.
+To run Jasmine tests in your command line (using Puppeteer), first install all necessary dependencies by executing `npm install`. After the `NPM` finishes fetching required modules, your workspace is ready to test.
 
+To run all tests from the monorepo's root directory, use the following command:
+* `npm run test`
```

Comment:
I'm not sure if this part is clear enough - `npm run test` run from the root directory is an alias to `npm run all test`, so it runs the tests for _all_ the packages.

This is probably what you meant by saying:

> To run individual tests (...)

but "individual tests" might suggest individual test cases, not package-specific tests, so this might need some more clarification.

Reply:
@jansiegel - as discussed, I rewrote the "Building" topic in 85825c5

Reply:
@jansiegel - as discussed, I rewrote the "Building" topic in 85825c5